vm_scripts/add_disk.py

```
@@ -0,0 +1,272 @@
+from pyVmomi import vim, vmodl
+from .helpers import VmHelper
+# test
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)